src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+  const obj = Object.entries(robot);
```

Comment:
`obj` is a really bad name for an array containing key, value pairs

Reply:
Yes, I'm used to naming so that my friends understand :D Correcting quickly